structure_tests/types/v2/metadata.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017 Google Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v2
+
+import (
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/runtimes-common/structure_tests/types/unversioned"
+)
+
+const CMD_PLACEHOLDER = "##METADATA_TEST_PLACEHOLDER"
```

Comment:
Can we leave CMD and entrypoint as nil if they're not specified, and use that as a sentinel instead of this placeholder?

Reply:
The reason I needed this is because I want the user to be able to specify a nil cmd/entrypoint in the config, but I also want them to be able to opt out of testing them if they want to. Unfortunately unmarshalling these to a struct both result in an empty list, so the code has no way of knowing which case we're dealing with. Initializing the struct with a placeholder creates a different case for nothing specified in the config vs. explicitly empty list in the config.